dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTrackerHelper.h

```diff
@@ -0,0 +1,60 @@
+﻿#pragma once
+#include <windows.h>
+#include <wil/result.h>
+#include <appmodel.h>
+#include <Microsoft.Windows.System.EnvironmentManager.h>
+
+using namespace winrt::Microsoft::Windows::System::implementation;
+
+static bool DoesChangeTrackingKeyExist()
+{
+    std::filesystem::path subKey = std::filesystem::path{ LR"(Software\Microsoft\AppModel\RegistryWriteVirtualization\ExcludedKeys\HKEY_CURRENT_USER/Software/ChangeTracker)" };
```

Comment:
NIT: `std::filesystem::path subKey{ LR"...`

`type-or-auto name{ value }` is your friend :-) Here `std::filesystem::path` is on both sides of the assignment. Rather redundant